src/main/webapp/app/overview/course-overview.component.html

```diff
@@ -1,55 +1,68 @@
 <div class="sidebar-wrapper">
     <div class="vw-100 bg-body">
         <mat-sidenav-container class="vw-100" [ngClass]="{ 'sidenav-height-dev': !isProduction || isTestServer, 'container-closed': isNavbarCollapsed }">
-            <mat-sidenav
-                #sidenav
-                disableClose
-                [ngClass]="{ 'sidenav-height-dev': !isProduction || isTestServer }"
-                class="module-bg rounded-end rounded-3"
-                opened="true"
-                mode="side"
-            >
+            <mat-sidenav disableClose [ngClass]="{ 'sidenav-height-dev': !isProduction || isTestServer }" class="module-bg rounded-end rounded-3" opened="true" mode="side">
                 <div class="sidebar-container d-flex h-100 justify-content-between flex-column" [ngClass]="{ collapsed: isNavbarCollapsed }">
                     <!-- NavItems -->
                     <div>
-                        <div class="d-flex p-3 align-items-center text-decoration-none" [title]="course?.title">
-                            @if (course && course.courseIcon) {
-                                <div>
-                                    <jhi-secured-image [src]="course.courseIcon" />
-                                </div>
-                            } @else {
-                                <div class="course-circle d-flex align-items-center justify-content-center">
-                                    <span class="fs-4">{{ course?.title | slice: 0 : 1 }}</span>
+                        <div id="container" class="d-flex p-3 align-items-center text-decoration-none" [title]="course?.title">
+                            <div ngbDropdown container="body" class="d-flex">
+                                @if (course && course.courseIcon) {
```

Comment:
```suggestion
                                @if (course?.courseIcon) {
```

Reply:
this does not work and leads to more errors :/